pkg/ddevapp/ddevapp.go

```diff
@@ -317,9 +317,12 @@ func (app *DdevApp) Describe(short bool) (map[string]interface{}, error) {
 				if svc, ok = v.(map[interface{}]interface{}); !ok {
 					continue
 				}
-
 				extraServices[serviceName] = map[string]string{}
 
+				extraServices[serviceName]["version"] = svc["image"].(string)
+
+				extraServices[serviceName]["status"], _ = dockerutil.GetContainerStateByName(svc["container_name"].(string))
+
```

Comment:
@rfay My new table also has the status for non-extra services. How would you like me to handle adding that information? Perhaps we should have a separate key for container statuses and set it before skipping the default containers?

Reply:
That sounds reasonable.